core/api/src/main/java/com/alipay/sofa/rpc/config/UserThreadPoolManager.java

```diff
@@ -76,4 +78,15 @@ public static UserThreadPool getUserThread(String service) {
         return userThreadMap == null ? null : userThreadMap.get(service);
     }
 
+    public static Map<String, UserThreadPool> getUserThreadPoolMap() {
```

Comment:
Where is this method used? Is it a read-only scenario?

Reply:
Right. It is used to monitor userThreadPool(currently only support monitoring default threadpool).

Reply:
Is hashcode meaningful to the user? or we could implement some unique identifier mechanism, e.g. if the same name is already present, then we append a counter after it?
> SofaUserProcessor-1, SofaUserProcessor-2